synapse/cryotank.py

```diff
@@ -207,52 +154,60 @@ async def info(self):
             dict: A dict containing items and metrics indexes.
         '''
         stat = self._items.stat()
-        return {'indx': self._items.index(), 'metrics': self._metrics.index(), 'stat': stat}
+        return {
+            'iden': self._iden,
+            'indx': self._items.index(),
+            'metrics': self._metrics.index(),
+            'stat': stat,
+        }
 
 class CryoApi(s_cell.CellApi):
     '''
     The CryoCell API as seen by a telepath proxy.
 
     This is the API to reference for remote CryoCell use.
     '''
+    async def _reqInit(self, name, conf=None):
+        if await self.cell.exists(name):
+            return await self.cell.init(name)
+
+        await self._reqUserAllowed(('cryo', 'tank', 'add'))
+        tank = await self.cell.init(name, conf=conf)
```

Comment:
if we passed an optional `User` to the cell.init() API we can do this permission check there + in the getCellApi case of initializing an empty tank by path.